README.md

```diff
@@ -136,8 +140,7 @@ npm start
 
 #### Running on Cloud development environments:
 
-- To run Remotion Preview or Renders, you need to set the server visibility to public, every time the server starts.
-  This is not recommended and must only be used in trusted environments.
```

Comment:
I think these are essential for the users to know if they wish to run this on GH codespaces. The command below does what is said in these lines 139-140